container/navigation.go

```diff
@@ -0,0 +1,196 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+// Navigation container is used to provide your application with a control bar and an area for content objects.
+// Objects can be any CanvasObject, and only the most recent one will be visible.
+//
+// Since: 2.6
+type Navigation struct {
+	widget.BaseWidget
+
+	Root  fyne.CanvasObject
+	Title string
+	Back  NavigationObject
+	Next  NavigationObject
```

Comment:
I think this likely adds more complexity over having the CreateRenderer just create buttons internally and let the developer wire up functions for what should happen on next and back. Just doing `OnBack func()` and `OnNext func()` will likely suffice. There are two options for that approach:

1. Always show backwards and forwards buttons and use a sensible approach for selecting what they do when no functions are set (just moving backwards and forwards).
2. Don't show buttons at all when functions are not set.